PubnativeLite/PubnativeLite/Banner/VWAdvertView.h

```diff
@@ -0,0 +1,63 @@
+//
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#import <CoreLocation/CoreLocation.h>
+#import "VWAdRequest.h"
+#import "HyBidAdSize.h"
+#import "HyBidAdView.h"
+#import "VWAdSize.h"
+
+@class VWAdvertView;
+
+@protocol VWAdvertViewDelegate
+
+- (void)advertViewDidReceiveAd:(nonnull VWAdvertView *)adView;
+
+- (void)advertView:(nonnull VWAdvertView *)adView didFailToReceiveAdWithError:(nullable NSError *)error;
+
+@end
+
+@interface VWAdvertView : UIView<HyBidAdViewDelegate>
+
+@property (nonatomic, strong) HyBidAdView * _Nonnull adView;
+
+@property (nonatomic, weak, nullable) id <VWAdvertViewDelegate, NSObject> delegate;
+
+@property (nonatomic, assign, readonly) BOOL adLoaded;
+
+@property (nonatomic, assign) VWAdSize adSize;
+
+- (nonnull instancetype)initWithSize:(VWAdSize)size;
+
+- (nonnull instancetype)initWithSize:(VWAdSize)size origin:(CGPoint)origin;
+
+- (void)loadRequest:(nonnull VWAdRequest *)request;
+
+- (CGSize)sizeThatFits:(CGSize)size;
+
+- (void)show;
```

Comment:
is this show method necessary? 🤔 (i'm asking because i did not see a method like this in VerveSDK.) if it was asilly question forgive me. 😄

Reply:
This we probably need because of the auto show on load feature on the banners we added for IQV

Reply:
Makes sense @eros902002. Good implementation then @faresbh 🥇

Reply:
I can't say here as I didn't receive a fill from Verve to test banners. 
But assuming they have a delegate to indicate when the banner is loaded than logically you will be able to show it there, and definitely provide a function for that.

Verve SDK, they don't have a show function, so that means it will be shown directly after finishing loading. 

I could hide this show function and move it when the delegate it's fired.